tests/it/c/compile/typeof_.d

```diff
@@ -0,0 +1,24 @@
+module it.c.compile.typeof_;
```

Comment:
I think this module should be called `extensions` instead of `typeof_`, since it's unlikely there'll be another test here.